tls/s2n_record_write.c

```diff
@@ -64,49 +64,57 @@ static int s2n_tls_record_overhead(struct s2n_connection *conn)
     return extra;
 }
 
-int s2n_record_rounded_write_payload_size(struct s2n_connection *conn, uint16_t size_without_overhead)
+/* Limits the size of the fragment length to the maximum allowed by the protocol */
+static int s2n_record_limit_write_payload_size(struct s2n_connection *conn, uint16_t size_without_overhead)
```

Comment:
Should this return a uint16_t?

Reply:
on 2nd thoughts, I think we could do away with s2n_record_limit_write_payload_size()

Reply:
Also if we're not using `conn` anymore probably best to remove it, yeah?